index.bs

```diff
@@ -306,20 +306,24 @@ The <dfn attribute for="DeviceOrientationEvent">absolute</dfn> attribute must re
 The <dfn method for="DeviceOrientationEvent">requestPermission(<var>absolute</var>)</dfn> method steps are:
 
  1. Let <var>global</var> be the <a>current global object</a>.
- 1. If <a>this</a>'s <a>relevant global object</a> does not have <a>transient activation</a>, return <a>a promise rejected with</a> a "{{NotAllowedError}}" {{DOMException}}.
+ 1. Let <var>hasTransientActivation</var> be true if <a>this</a>'s <a>relevant global object</a> has <a>transient activation</a>, and false otherwise.
  1. Let <var>result</var> be <a>a new promise</a> in <a>this</a>'s <a>relevant Realm</a>.
  1. Run these steps <a>in parallel</a>:
      1. If <var>absolute</var> is true:
          1. Let <var>permissions</var> be « "<a permission>accelerometer</a>", "<a permission>gyroscope</a>", "<a permission>magnetometer</a>" ».
      1. Otherwise:
          1. Let <var>permissions</var> be « "<a permission>accelerometer</a>", "<a permission>gyroscope</a>" ».
-     1. Let <var>permissionState</var> be "<a for="permission">granted</a>".
+     1. <a for="list">For each</a> <var>name</var> of <var>permissions</var>:
+         1. If <var>name</var>'s <a>permission state</a> is "{{PermissionState/prompt}}" and <var>hasTransientActivation</var> is false:
+             1. <a>Queue a global task</a> on the <a>device motion and orientation task source</a> given <var>global</var> to <a>reject</a> <var>result</var> with a "{{NotAllowedError}}" {{DOMException}}.
```

Comment:
I would probably add a comma before "to reject"

Reply:
Do you mean _"Queue a global task, to [...]"_? Isn't it usual to just say "queue a task to xyz"?